holmes-py/.python-version

```diff
@@ -1 +1 @@
-3.9.4
+3.12.1
```

Comment:
Do we want this to be `3.12` allowing users to use any flavor of 3.12.x or more strictly `3.12.1`?